config/settings.yml

```diff
@@ -8,9 +8,10 @@ DOR_SERVICES:
   URL: 'http://localhost:3003'
   TOKEN: 'secret-token'
 
-
 workflow_url: 'https://workflow-env.stanford.edu/workflow'
 workflow:
   logfile: 'log/workflow_service.log'
   shift_age: 'weekly'
   timeout: 60
+
+exp_worker_count: 5 # for okcomputer endpoint
```

Comment:
Would you mind spelling this out as `expected_worker_count`?

Reply:
np;  just pushed this change.